pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -70,7 +70,7 @@ func Register(ctx context.Context, management *config.ManagementContext) {
 }
 
 func (c *Controller) Create(b *v3.EtcdBackup) (runtime.Object, error) {
-	if v3.BackupConditionCompleted.IsFalse(b) {
+	if v3.BackupConditionCompleted.IsFalse(b) || v3.BackupConditionCompleted.IsTrue(b) {
```

Comment:
what else can it be besides true or false?

Reply:
The condition has 3 values:
- true: the backup succeeds
- false: the backup failed,
- unknown: the backup is still in progress

However, the backup object is initially created with the condition unset. So, we need to check for specific values.

Reply:
Ideally we should create a backup with Unknown condition set from the beginning, but it's ok for now.